plaid/requester.py

```diff
@@ -20,12 +20,25 @@ def _requests_http_request(url, method, data, timeout=DEFAULT_TIMEOUT):
             headers={
                 'User-Agent': 'Plaid Python v{}'.format(__version__),
             },
-            timeout=timeout,
-        )
+            timeout=timeout, )
     else:
-        raise Exception(
-            'Invalid request method {}'.format(method)
-        )
+        raise Exception('Invalid request method {}'.format(method))
+
+
+async def _requests_http_request_async(url,
+                                       method=None,
+                                       data=None,
+                                       timeout=DEFAULT_TIMEOUT):
+    import aiohttp
+    headers = {'content-type': 'application/json'}
+    async with aiohttp.ClientSession(read_timeout=timeout) as session:
```

Comment:
`aiohttp` seems to generally recommend using a single client session over an application's lifetime (http://aiohttp.readthedocs.io/en/stable/client_reference.html). 

What would you think of creating a default session object at the top of the file, and then passing `timeout` as a parameter into `session.post()` instead?